pkg/security/util.go

```diff
@@ -2,6 +2,7 @@ package security
 
 const (
 	UIDRangeAnnotation     = "openshift.io/sa.scc.uid-range"
+	FSGroupAnnotation      = "openshift.io/sa.scc.fsgroup"
```

Comment:
I think we need to differentiate FSGroup from the allowed supplemental groups.  There are use-cases where the space of reserved GIDs for FSGroup is going to be separate from the space used for shared storage.